Source/WebCore/css/CSSCounterStyleDescriptors.cpp

```diff
@@ -67,6 +67,27 @@ static String symbolToString(const CSSValue* value)
     return primitiveValue.stringValue();
 }
 
+static CSSCounterStyleDescriptors::AdditiveSymbols translateAdditiveSymbolsFromStyleProperties(const StyleProperties& properties)
+{
+    auto additiveSymbols = properties.getPropertyCSSValue(CSSPropertyAdditiveSymbols).get();
+    if (!is<CSSValueList>(additiveSymbols))
+        return { };
+    auto& list = downcast<CSSValueList>(*additiveSymbols);
+    CSSCounterStyleDescriptors::AdditiveSymbols result;
+    for (auto& additiveValue : list) {
+        if (auto* additiveTuple = dynamicDowncast<CSSValuePair>(&additiveValue)) {
+            auto& weightValue = additiveTuple->first();
+            auto& symbolValue = additiveTuple->second();
+            ASSERT(weightValue.isPrimitiveValue() && symbolValue.isPrimitiveValue());
+            auto& weight = downcast<CSSPrimitiveValue>(weightValue);
+            ASSERT(weight.isInteger());
```

Comment:
If it is not an integer the parser should have invalidated the rule before the translation but I thought it would be good to assert here to make sure.

Reply:
I think we can remove this assert in that case.

Reply:
I think it’s good to have assertions that the code actually relies on, and that often happens naturally due to assertions built into things like `downcast`, but here we just rely on being able to **convert** to an integer, so it’s probably overkill to additionally assert.